### PR TITLE
[ui][fix] Workaround firefox file upload

### PR DESCRIPTION
**Pull request by asmigala**
//test: `@create-api-connector-from-file-test or @extension-CRUD-import-new or @import-edit-integration`

### CODE REVIEW

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by asmigala** [comment]
Firefox test run: https://fuse-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/syndesis-tests-firefox/192/Cucumber_20report_20-_20firefox/

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

File: ui-tests/src/test/java/io/syndesis/qe/utils/UploadFile.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+     * Firefox 60 ESR does not allow sending keys to hidden file input element
+     * (see https://bugzilla.mozilla.org/show_bug.cgi?id=1502864 )
+     * the easiest way to workaround is to remove display: none style from the input
+     *
+     * @param selector the input element selector to upload the file to. Will be made visible on firefox
+     * @param file file to upload
+     */
+    public static void uploadFile(By selector, File file) {
+        uploadFile($(selector), file);
+
```
Comment (mcada) [inline ui-tests/src/test/java/io/syndesis/qe/utils/UploadFile.java]: should this empty line be here?

Comment (asmigala) [reply ui-tests/src/test/java/io/syndesis/qe/utils/UploadFile.java]: Removed

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Pull request merged**